packages/fxa-auth-server/fxa-oauth-server/test/api.js

```diff
@@ -3410,10 +3410,10 @@ describe('/v1', function() {
           .then(function (res) {
             var result = res.result;
             assert.equal(result.length, 2);
-            assert.equal(result[0].id, client1Id.toString('hex'));
-            assert.deepEqual(result[0].scope, ['clients:write', 'profile']);
-            assert.equal(result[1].id, client2Id.toString('hex'));
-            assert.deepEqual(result[1].scope, ['scopeFromRefreshToken']);
+            assert.equal(result[0].id, client2Id.toString('hex'));
```

Comment:
Sorry this deserved a note - there was a timing issue with this test that was causing it to fail locally for me but usually pass in CI, because the output gets sorted by timestamp and then name. Changing the name above forced it into a consistent order regardless of timestamp, and that order happens to be the reverse of what was there previously.